fe/fe-core/src/main/java/org/apache/doris/catalog/CatalogRecycleBin.java

```diff
@@ -112,7 +112,7 @@ public synchronized boolean allTabletsInRecycledStatus(List<Long> backendTabletI
     }
 
     private void addRecycledTabletsForTable(Set<Long> recycledTabletSet, Table table) {
-        if (table.getType() == TableType.OLAP) {
```

Comment:
could we restore async mv from recycle bin? or we just mv it into recycle bin when we drop it?